services/app/src/components/views/Tools.tsx

```diff
@@ -12,50 +10,27 @@ export interface StateProps {
 
 export interface DispatchProps {
   onCustomCombatSelect: (settings: SettingsType) => void;
-  onQuestCreatorSelect: () => void;
   onPrivateQuestsSelect: (settings: SettingsType, user: UserState) => void;
-  onMultiplayerSelect: (user: UserState) => void;
-  testMusic: () => void;
 }
 
 export interface Props extends StateProps, DispatchProps {}
 
 const Tools = (props: Props): JSX.Element => {
   return (
     <Card title="Tools">
+      <p>(multiplayer moved to player / game mode select menu - double tap on the home screen to access)</p>
```

Comment:
"player / game mode select" -> "game mode select". and maybe move this to the bottom of the card + deemphasize?